src/borg/archiver.py

```diff
@@ -1307,8 +1309,11 @@ def do_debug_dump_manifest(self, args, repository, manifest, key):
 
         meta = self.debug_json(msgpack.fallback.unpackb(data, object_hook=StableDict, unicode_errors='surrogateescape'))
 
-        with open(args.path, 'w') as fd:
-            json.dump(meta, fd, indent=4)
+        if args.path == '-':
+            json.dump(meta, sys.stdout, indent=4)
+        else:
+            with open(args.path, 'w') as fd:
```

Comment:
maybe we could (later?) refactor this to some dash_open(path, mode) context manager that handles "opening" stdin/out and fs files (and also closing, if needed) to avoid this duplication (likely also found at some other places).